hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -810,6 +810,9 @@ def _mask_object(object_to_be_masked, topfile):
             column = blacklisted_object['column']  # Can be converted to list as well in future if need be
             # Name of column that stores environment variables
             custom_mask_column = blacklisted_object.get('custom_mask_column', '')
+            enable_local_masking = blacklisted_object.get('enable_local_masking', '').lower()
```

Comment:
Can we please use the normal `None` default for these `.get()` calls? I'd like to use actual booleans where possible, since

```
if var is True:
```

is substantially faster than

```
if var == True:
```

and *way* faster than

```
if var == 'true':
```